docs-2.0/3.ngql-guide/7.general-query-statements/3.go.md

```diff
@@ -52,7 +48,9 @@ OVER <edge_type_list> [{REVERSELY | BIDIRECT}]
 
         There are some restrictions for the `WHERE` clause when you traverse along with multiple edge types. For example, `WHERE edge1.prop1 > edge2.prop2` is not supported.
 
-- `YIELD [DISTINCT] <return_list>`: specifies the desired output. For more information, see [YIELD](../8.clauses-and-options/yield.md). When not specified, the destination vertex IDs will be returned by default.
+- `YIELD [DISTINCT] <return_list>`: defines the output to be returned. It is recommended to use the [Schema function](../6.functions-and-expressions/4.schema.md) to fill in `<return_list>`. `src(edge)`, `dst(edge)`, `type(edge) )`, `rank(edge)`, `properties(edge)`, `id(vertex)`, and `properties(vertex)` are currently supported, while nested functions are not. For more information, see [YIELD](../8.clauses-and-options/yield.md). When not specified, the destination vertex IDs will be returned by default.
```

Comment:
[When not specified, the destination vertex IDs will be returned by default.]
is it better to add [all edges] or [edges] at the end of [the destination vertex IDs] if the word "the" is used?